src/Akeneo/Tool/Bundle/MeasureBundle/Resources/public/pages/edit/Edit.tsx

```diff
@@ -267,7 +267,7 @@ const Edit = () => {
         buttons={buttons}
         breadcrumb={
           <Breadcrumb>
-            <Breadcrumb.Step href={settingsHref}>{translate('pim_menu.tab.settings')}</Breadcrumb.Step>
+            <Breadcrumb.Step href={`#${settingsHref}`}>{translate('pim_menu.tab.settings')}</Breadcrumb.Step>
```

Comment:
As you are on edition page, could you use the `onClick` props with `router.redirect` action instead of the href? Using router with redirect should force the user to confirm before changing page if there are unsaved changes.
```suggestion
            <Breadcrumb.Step onClick={() => router.redirect(settingsHref)}>{translate('pim_menu.tab.settings')}</Breadcrumb.Step>
```